packages/davinci-client/src/lib/config.types.ts

```diff
@@ -16,7 +16,7 @@ export interface InternalDaVinciConfig extends DaVinciConfig {
  * DaVinci configuration options that extends the Forgerock SDK configuration options
  */
 // export type InternalDaVinciConfig =
-//   | ((DavinciConfigWithResponseType | AsyncConfigOptions) & {
+//   | ((DaVinciConfigWithResponseType | AsyncConfigOptions) & {
 //       openIdConfiguration: OpenIdResponse;
 //     })
-//   | (DavinciConfigWithResponseType | AsyncConfigOptions);
+//   | (DaVinciConfigWithResponseType | AsyncConfigOptions);
```

Comment:
If we no longer need these, let's remove them. I commented them out as I wasn't sure if I was going to need them or not.